<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SynchronizeRL</class>
 <widget class="QWidget" name="SynchronizeRL">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>782</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="grbProject">
     <property name="title">
      <string>Project</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Note:&lt;/span&gt;&lt;/p&gt;&lt;p&gt;1.Please make sure the project is right.&lt;/p&gt;&lt;p&gt;2.You can change the project at RL project interface.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Current Project</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edtProject"/>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="grbSyncRL">
     <property name="title">
      <string>Export To RL Project</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="btnExport">
          <property name="text">
           <string>Export All</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnExportWorkStaPoint">
          <property name="text">
           <string>Export WorkStaPoint</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnExportBoatPoint">
          <property name="text">
           <string>Export BoatPoint</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnExportBacketPoint">
          <property name="text">
           <string>Export BasketPoint</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnExportToolWobj">
          <property name="text">
           <string>Export Tool&amp;&amp;Wobj</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnClear">
          <property name="text">
           <string>Clear Log</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="plainTextEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
